Tasks/SqlAzureDacpacDeployment/task.json

```diff
@@ -36,7 +36,7 @@
 	"inputs": [
 		{
 			"name": "ConnectedServiceName",
-			"type": "connectedService:Azure",
+			"type": "connectedService:Azure:Certificate,UsernamePassword",
```

Comment:
Azure SQL currently does not work with SPN based credentials. We had started a mail thread with Azure team, but got no reply. Hence using only certificate/credentials as of now.